service/controller/resource/alerting/ingress/v1beta1/test/case-1-awsconfig.golden

```diff
@@ -0,0 +1,25 @@
+apiVersion: v1beta1
+kind: Ingress
+metadata:
+  annotations:
+    kubernetes.io/ingress.class: nginx
+    nginx.ingress.kubernetes.io/auth-signin: https://$host/oauth2/start?rd=$escaped_request_uri
+    nginx.ingress.kubernetes.io/auth-url: https://$host/oauth2/auth
+  creationTimestamp: null
+  labels:
+    app.kubernetes.io/instance: alertmanager
+    app.kubernetes.io/managed-by: prometheus-meta-operator
+    app.kubernetes.io/name: alertmanager
+  name: alertmanager
+  namespace: alice-prometheus
+spec:
+  rules:
+  - host: https://prometheus
```

Comment:
This tells me something is not quite right.

Reply:
It's fixed